app/views/templates/search/channels_landing.rb

```diff
@@ -56,486 +34,482 @@ def navigation
-                  :url   => "#777"
+                  text: "Deutsche National Bibliothek",
+                  url: "#777"
                 },
                 {
-                  :text  => "Netherlands institute for sound and vision",
-                  :url   => "#777"
+                  text: "Netherlands institute for sound and vision",
+                  url: "#777"
                 },
                 {
-                  :text => "Cité de la Musique",
-                  :url => "#777"
+                  text: "Cité de la Musique",
+                  url: "#777"
                 }
               ]
             }
           },
-          :hero_config =>{
-            :hero_image => "sample/channel_hero_music.jpg",
-            :attribution_text => "This is a placeholder image found somewhere",
-            :attribution_url => "http://europeana.eu",
-           # :license_CC-ND-NC-SA => true
-            :license_CC_ND_NC_SA => true
+          hero_config: {
+            hero_image: "sample/channel_hero_music.jpg",
+            attribution_text: "This is a placeholder image found somewhere",
+            attribution_url: "http://europeana.eu",
+           # license_CC-ND-NC-SA: true
+            license_CC_ND_NC_SA: true
           },
-          :channel_entry =>{
-            :title => "Promoted title",
-            :items =>[
+          channel_entry: {
+            title: "Promoted title",
+            items: [
               {
-                :title =>"Erlkonig",
-                :url => "urlhere",
-                :url => root_url + "channels/music?q=erlkonig",
-                :count => "12",
-                :media_type =>"audio recordings"
+                title: "Erlkonig",
+                url: "urlhere",
+                url: root_url + "channels/music?q=erlkonig",
+                count: "12",
+                media_type: "audio recordings"
               },
               {
-                :title => "Mahler 5",
-                :url => root_url + "channels/music?q=mahler",
-                :count =>  "15",
-                :media_type => "video recordings"
+                title: "Mahler 5",
+                url: root_url + "channels/music?q=mahler",
+                count: "15",
+                media_type: "video recordings"
               },
               {
-                :title =>"Beethoven’s handwriting",
-                :url => root_url + "channels/music?q=beethoven+writing",
-                :count => "2",
-                :media_type =>"scores"
+                title: "Beethoven’s handwriting",
+                url: root_url + "channels/music?q=beethoven+writing",
+                count: "2",
+                media_type: "scores"
               },
               {
-                :title =>"Haitink in Berlin",
-                :url => root_url + "channels/music?q=Haitink",
-                :count => "12",
-                :media_type =>"letters"
+                title: "Haitink in Berlin",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.